README.rst

```diff
@@ -1,4 +1,39 @@
-Here there will be documentation for the generator
+Sphinx GitHub Pages Generator
+*****************************
 
-..
-   comment #TODO
\ No newline at end of file
+.. _docu_start:
+
+Overview
+========
+
+This project provides a python library for accessing automatic generation of sphinx GitHub pages 
+for a given repository. It is intended for th use with a Continuous Integration solutions.
+Developed using GitHub Actions.
+
+In a Nutshell
+=============
+
+Prerequisites
+-------------
+
+- Python 3.8+
+
+Installation
+-------------
+#todo fix link
+Install the package from Github via `pip`:: #todo make pip package
+
+    pip install -e git://github.com/exasol/bucketfs-utils-python.git@{tag name}#egg=exasol-bucketfs-utils-python
```

Comment:
I think we should encourage the clients to use the wheel package, and not building the package from Git.

Reply:
Yes, but i have never done a release and don't know exactly how the installation works. this here is just a placeholder for now..